js/templates/transactionModal.html

```diff
@@ -993,7 +993,7 @@ <h3 class="txt-bright"><%= polyglot.t('Loading') %> </h3>
         <div>
           <input type="checkbox" class="fieldItem"
                  id="transactionsCloseDisputeCheckbox" >
-          <label for="transactionsCloseDisputeCheckbox" class="fontSize10 clickable compact">
+          <label for="transactionsCloseDisputeCheckbox" class="fontSize10 clickable compact" style="white-space: nowrap;">
```

Comment:
This phrase could be various lengths when translated, is there a solutions that would allow it to wrap so very long translations will still be readable?